testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -4904,6 +5012,87 @@ protected void verifyConcurrentAddColumnFailurePermissible(Exception e)
         throw new AssertionError("Unexpected concurrent add column failure", e);
     }
 
+    // Repeat test with invocationCount for better test coverage, since the tested aspect is inherently non-deterministic.
+    @Test(timeOut = 60_000, invocationCount = 4)
+    public void testCreateOrReplaceTableConcurrently()
+            throws Exception
+    {
+        if (!hasBehavior(SUPPORTS_CREATE_OR_REPLACE_TABLE)) {
+            // Already handled in testCreateOrReplaceTableWhenTableNotYetExists
```

Comment:
The test `testCreateOrReplaceTableWhenTableNotYetExists` seems not to exist anymore.